builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -455,6 +455,168 @@ func TestAccVSphereVirtualMachine_createWithCdrom(t *testing.T) {
 	})
 }
 
+func TestAccVSphereVirtualMachine_updateMemory(t *testing.T) {
+	var vm virtualMachine
+	var locationOpt string
+	var datastoreOpt string
+
+	if v := os.Getenv("VSPHERE_DATACENTER"); v != "" {
```

Comment:
If these are the standard env vars that are required to run the resources outside of the tests, then they are not needed here - they are taken care of in provider_test.go